src/main/java/tr/edu/ogu/ceng/model/InternshipRegistry.java

```diff
@@ -0,0 +1,46 @@
+package tr.edu.ogu.ceng.model;
+
+import java.sql.Timestamp;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name="ims_internship_registries")
+public class InternshipRegistry {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "file_path", nullable = false)
+    private String file_path;
+
+    @Column(name = "name", nullable = false)
+    private String name;
+
+    @Column(name = "type", nullable = false)
+    private String type;
+
+    @Column(name = "date", nullable = false)
+    private Timestamp date;
+
+    @Column(name = "create_date", nullable = true)
+    private Timestamp create_date;
+
+    @Column(name = "update_date", nullable = true)
+    private Timestamp update_date;
+
+    @Column(name = "internship_id", nullable = false)
+    private Long internship_id;
```

Comment:
JoinColumn ilişkisi kurulmalı sanki.

Reply:
Hocam birden fazla tablo olduğu için daha onlar projeye eklenmeden yapmak istemedim. Burada internship entitiy sınıfını oluşturmadan JoinColumn yaparsam hata vermez mi?

Reply:
Bu sınıfı eklemelisin bu durumda. Hatalı bir şekilde projeyi göndermemelisin @Sdrs81

Reply:
@ykartal hocam herhangi bir geri bildirimde bulunmadınız son yorumum ile ilgili.

Reply:
Ne gerekiyorsa yapılmalı, yani oluşturman gerekiyorsa oluşturursun belki temel olarak belki tam olarak ama oluşturursun başkası üzerinde çalışıyor olabilir sonradan uygun şekilde birleşecektir geliştirilen kodlar.

Reply:
@ykartal hocam istediğiniz değişikliği yaptım. Internship entity sınıfını ekledim ve InternshipRegistry entity sınıfını da bunun için düzenledim. Bir yanlışlık var mı şuan acaba?